Base/res/html/misc/navigator-beacon.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Navigator Beacon</title>
+</head>
+<body>
+    <div id="log"></div>
+    <script>
+        const log = document.getElementById('log');
+        log.textContent = navigator.sendBeacon('https://serenityos.org/') ? 'Successfully send beacon' : 'Failed to send beacon';
```

Comment:
Can we make this an actual test? (maybe with some failure cases? :) )

Reply:
Sure I will add a text test